src/compile/axis.ts

```diff
@@ -295,6 +325,41 @@ export namespace properties {
       }
     }
 
+    if (axis.tickLabelColor !== undefined) {
+        labelsSpec.stroke = {value: axis.tickLabelColor};
+    }
+
+    if (axis.tickLabelFont !== undefined) {
+        labelsSpec.font = {value: axis.tickLabelFont};
+    }
+
+    if (axis.tickLabelFontSize !== undefined) {
+        labelsSpec.fontSize = {value: axis.tickLabelFontSize};
+    }
+
     return labelsSpec || undefined;
   }
+
+  export function ticks(model: Model, channel: Channel, ticksPropsSpec, def) {
```

Comment:
@YuhanLu 

let's remove `def` from all of these methods if it's not used.  (If it's used like `labels()`, we can leave it in.)